README.md

```diff
@@ -15,47 +15,99 @@ This document describes the protocol to compress Verifiable Credentials into URI
+    1. Data minimalism by requirements of the application
+    1. Allows for the use of feature phones and low energy chips as scanners. 
+    1. It is ideal size and character set to use on SMS, SmartCards and NFC tags
+    1. It's readable by any text file
+1. Freedom of payload specification
+    1. Any new payload can be described and approved in minutes
+    1. Business logic is part of the payload definition as opposed to hidden
+1. Freedom of cryptographic tools to sign for the packages. 
+    1. RSA, ECDSA, JWTs and other signature models are supported
+    1. Protocol guarantees easily accessible Public Keys by design
+    1. Public Key files describe the algorithms being used by the issuer
+1. Composability of QRs enable privacy and selective disclosure
+    1. The hash of one QR can be used in another, building a signed chain of QRs
+1. Domain-based trust model
+    1. If users trust `cdc.gov`, they will also trust `keys.cdc.gov` 
+    1. There is no need for a centralized issuing authority
+    1. No need to trust app developers or private lists of approved issuers
+1. Enhanced Security
+    1. The signed payload is cryptographically protected and thus impossible to tamper
+1. Freedom of binding
+    1. There's no requirement to bind certificate to a user or proxy models to a user
+    1. Electronic binding with individual wallets is possible when the certificate is loaded on an app. 
+1. Extremely private  
+    1. The only information online is a public key.
+    1. The protocol does not require centralized servers. 
+    1. No exposure to government, no blockchain, no private companies, no trusted lists
+    1. There's no need to manage did's or other private issuing systems
+1. Easy to backup 
+    1. A simple picture or a paper copy serves as a backup of the code
+1. Negligible cost per user on all activities
+1. Generalizable: 
+    1. Any record/payload can be created and signed in the same format
+1. Free as in beer
+    1. The entire specification is licensed under MIT License
+
+Disadvantages are: 
+1. Traceability of the QR Codes is possible by colluding verifiers
+    1. Solutions include generating multiple salted QR Codes to be given away as opposed to one code that is read everywhere
+1. Chance of losing the QR, losing the data
+    1. The issuer might have a copy of the event record, but it is not a requirement
+    1. Apps and pictures of the card can serve as a backup
+1. No Real Time Revocation of cards/credentials 
+    1. Issuer removes the public key from the domain, invalidating all certificates
+    1. The verifier app needs to wait for the next issuer list (or revocation list) update
+1. The information in the QR itself is not encrypted by design
+    1. Payload specifications can include password-protected fields
+1. Fields are case insensitive by design
+    1. Payloads are encoraged to encode each field in Base32 to support case sensitivity
```

Comment:
I'd add something like "If case sensitivity is required," because we'd really prefer not to base32 every string due to size

Reply:
fixed